samples/asgardeo-react-app/src/pages/IssuerClaimValidationFailuer.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FunctionComponent, ReactElement } from "react";
+import { DefaultLayout } from "../layouts/default";
+
+/**
+ * Page to display for ID token claim validation failure.
+ *
+ * @return {React.ReactElement}
+ */
+export const IssuerClaimValidationFailure: FunctionComponent = (): ReactElement => {
+
+    return (
+        <DefaultLayout>
+            <h6 style={{
```

Comment:
Let's use CSS classes instead of in-line styling. Check in other places as well

Reply:
fixed in commit https://github.com/asgardeo/asgardeo-auth-react-sdk/commit/2ded725d7c96494fedec39cb54bdd58556460ce6